README.md

```diff
@@ -1,24 +1,28 @@
-# ArmA 3 Unix Launcher
+# Arma 3 + DayZ SA Unix Launcher
 
 [![Codacy Badge](https://api.codacy.com/project/badge/Grade/8a144e12d9cc4cde90616f0e3f282322)](https://www.codacy.com/manual/muttleyxd/arma3-unix-launcher?utm_source=github.com&amp;utm_medium=referral&amp;utm_content=muttleyxd/arma3-unix-launcher&amp;utm_campaign=Badge_Grade) [![Build Status](https://img.shields.io/drone/build/muttleyxd/arma3-unix-launcher?label=Linux%20build&logo=drone)](https://cloud.drone.io/muttleyxd/arma3-unix-launcher) [![Actions Status](https://img.shields.io/github/workflow/status/muttleyxd/arma3-unix-launcher/Mac%20OS%20X%20release%20deployment/master?label=Mac%20OS%20X%20build)](https://github.com/muttleyxd/arma3-unix-launcher/actions)
 
-ArmA 3 Launcher for Linux and Mac.
-Since Bohemia didn't port their launcher to Linux and Mac and existing launcher didn't satisfy my needs I decided to create my own.
+A clean, intuitive Arma 3 + DayZ SA Launcher for GNU/Linux and MacOS.
+
+Since Bohemia Interactive's developers have not ported the official Arma 3 and DayZ Launchers to GNU/Linux and MacOS, and the existing Steam Proton launch options did not satisfy the community's needs, I decided to create a new one. The Arma 3 + DayZ SA Unix Launcher is intuitive, working with all of Bohemia Interactive's launch parameters for each title and integrating neatly with mod subscriptions in the Steam Workshop.
 
 ## Table of contents
 
 * [Installing](#installing)
     * [From package](#from-package)
     * [Building from source](#building-from-source)
+      * [Arma 3](#build-process-for-arma-3)
+      * [DayZ](#build-process-for-dayz)
 * [Launch parameters](#launch-parameters)
 * [Sceenshots](#screenshots)
+* [DayZ Installation Tips](#dayz-installation)
 
 ## Installing
 ### From package
 
-For Debian based distributions (Debian, Ubuntu), Arch based distributions (Arch, Manjaro) and Mac OS X there are packages available in [releases tab!](https://github.com/muttleyxd/arma3-unix-launcher/releases)
+For Debian based distributions (Debian, Ubuntu), Arch based distributions (Arch, Manjaro), and MacOS there are packages available in [releases tab!](https://github.com/muttleyxd/arma3-unix-launcher/releases). Choose either Arma 3 or DayZ.
```

Comment:
If we're finally updating this I would mention AppImage as it works anywhere where we have GNU/Linux.

```
There are GNU/Linux and macOS packages available in [releases tab!](https://github.com/muttleyxd/arma3-unix-launcher/releases).
For GNU/Linux following package types are available:
- pkg.tar.xz package (for Arch Linux, Manjaro and few others) (arma3-unix-launcher)
- deb package for Ubuntu 16.04 and 18.04 (arma3-unix-launcher)
- dmg package for macOS (arma3-unix-launcher)
- AppImage packages for any GNU/Linux running glibc 2.23 or newer (arma3-unix-launcher and dayz-linux-launcher)
```